README.md

```diff
@@ -4,6 +4,6 @@
 2. ```vagrant up```
 3. ```vagrant ssh```
 4. ```cd /project_data/```
```

Comment:
Yevhen, 'project_data' name does not linked to any project name and was added when we did not have any project template. 
However, you can rename it in whatever you want in Vagrantfile (please write it to Andrey, and he will update file).